projects/croniter/fuzz_iter.py

```diff
@@ -17,32 +17,32 @@
 import sys
 
 from datetime import datetime
-import croniter
 
+import croniter
 
-@atheris.instrument_func
 def TestOneInput(data):
   fdp = atheris.FuzzedDataProvider(data)
   base = datetime(2012, 4, 6, 13, 26, 10)
   try:
-    itr = croniter.croniter(fdp.ConsumeString(50), base)
+    cron_str = fdp.ConsumeString(50)
+    hash_id = fdp.ConsumeBytes(2)
+    croniter.croniter.is_valid(cron_str)
+    itr = croniter.croniter(cron_str, base, hash_id=hash_id)
     idx = 0
     for v in itr.all_next():
       idx += 1
       if idx > 10:
         break
+    itr.get_next(base)
+    itr.get_prev(base)
   except (croniter.CroniterBadCronError, croniter.CroniterBadDateError) as e:
     pass
   except NameError as e:
     # Catch https://github.com/kiorky/croniter/blob/bb5a45196e5f8f15fd0890f4ee5e9697671a3fe2/src/croniter/croniter.py#L781
     if not "'exc' is not defined" in str(e):
       raise e
 
-
-def main():
```

Comment:
I'd probably prefer to keep main functions -- any particular reason to change the style?

Reply:
I don't, I've tidied up the `fuzz_iter.py` to fix all the unneeded changes (whitespace, def main removed etc).